src/components/PlayClassCard/PlayClassCard.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import "../PlayClassCard/PlayClassCard.css";
+import { IoPlay, IoStop, IoPause, IoHeart } from "react-icons/io5";
+import { useState } from "react";
+
+export default function PlayClassCard(props) {
+  const [isFavourite, setIsFavourite] = useState("false");
+  const [isActive, setIsActive] = useState(0);
+
+  const {
+    name,
+    isToggled,
+    isPlaying,
+    isStopped,
+    isPaused,
+    prepTime,
+    duration,
+    ...rest
+  } = props;
+
+  const toggleHeartIcon = () => {
+    setIsFavourite(!isFavourite);
+  };
+
+  function onPauseClick() {
+    setIsActive(1);
+  }
+
+  function onPlayClick() {
+    setIsActive(2);
+  }
+
+  function onStopClick() {
+    setIsActive(3);
+  }
+
+  return (
+    <section className="PlayClass" {...rest}>
+      <IoHeart
+        className={isFavourite ? "FavIcon--inactive" : "FavIcon--active"}
+        onClick={toggleHeartIcon}
+      />
+      <h2>{name}</h2>
+      <div className="PlayClass__controls">
+        <IoPause
+          onClick={onPauseClick}
+          className={
+            isActive === 1
+              ? "PlayClass__controls--active"
+              : "PlayClass__controls--inActive"
+          }
+        />
+        <IoPlay
+          onClick={onPlayClick}
+          className={
+            isActive === 2
+              ? "PlayClass__controls--active"
+              : "PlayClass__controls--inActive"
+          }
+        />
+        <IoStop
+          onClick={onStopClick}
+          className={
+            isActive === 3
+              ? "PlayClass__controls--active"
+              : "PlayClass__controls--inActive"
+          }
+        />
+      </div>
+      <h3>Preparation Time</h3>
+      <div className="PrepTime">
+        <div className="PrepTime__line"></div>
+        <div className="PrepTime__Line--time">
+          <p>0</p>
+          <p>{prepTime}</p>
+        </div>
+      </div>
+      <h3>Class</h3>
```

Comment:
I think a `h4` would be better because it's in the same hierarchy as the other `h` tag.

Reply:
They should actually have the same weight. PrepTime and ClassTime.

Reply:
You would solve this with CSS. I was talking semantically.